src/content/docs/usage/Node Graph/Tutorials/your-first-graph.mdx

```diff
@@ -11,69 +11,94 @@ import { Steps } from '@astrojs/starlight/components';
 import { Image } from 'astro:assets';
 import ctxMenuImage from './assets/ctx-menu.png';
 import { Quiz, QuizOption } from 'starlight-videos/components'
+import searchNodes from '.assets/search-for-nodes';
```

Comment:
Asset path is wrong as doc is unable to load. You probably meant `./assets/search-for-nodes.png`